README.md

```diff
@@ -1,3 +1,71 @@
 # To Do List
 
-A To-do list is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete, delete them or reshuffle them.
+> The To-do list project is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete, or delete them.
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript (ES6)
+- Linters
+- Webpack
+
+
+## Live Demo
+[Checkout here](https://julzedz.github.io/to-do-list/)
```

Comment:
Kindly check this address because it is not showing the correct address for your deployment project. (remember to always add the "dist" folder)

Reply:
Hi, @mgmediaweb Kindly check out this commit (`Update html2 [93b0d9e]`). That's my project. Had to merge with the main branch to resolve some conflicts. That's why you're seeing errors. 🙏